ui/app/templates/components/configure-aws-secret.hbs

```diff
@@ -61,6 +45,7 @@
       (hash access_key=@accessKey iam_endpoint=@iamEndpoint sts_endpoint=@stsEndpoint secret_key=@secretKey region=@region)
     }}
     data-test-aws-root-creds-form="true"
+    aria-label="save root creds form"
```

Comment:
helps with linting warning about two form element being on the same page that need to be uniquely identified.